SpecFlowTask/SpecFlowTask/StepDefinitions/LangSkillStepDefinitions.cs

```diff
@@ -33,9 +35,24 @@ public void ThenTheLanguageShouldBeAddedSuccessfully()
             LanguageAndSkillsPage lanAndSkillPageObj = new LanguageAndSkillsPage();
 
             string NewLanguage = lanAndSkillPageObj.GetLanguage(driver);
-            Assert.That(lanAndSkillPageObj.GetLanguage(driver) == "Italian");
+            Assert.That(lanAndSkillPageObj.GetLanguage(driver) == "Sinhala");
+        }
+
+        [When(@"I add multiple '([^']*)' and '([^']*)' to the Languages")]
+        public void WhenIAddMultipleAndToTheLanguages(string language, string level)
+        {
+            LanguageAndSkillsPage lanAndSkillPageObj = new LanguageAndSkillsPage();
+            lanAndSkillPageObj.AddMultipleLanguage(driver, language, level);
```

Comment:
can you stop passing the driver driver around to the methods as discussed previously 


Can you please extend to common driver or pass the driver when you initialise the page, avoid passing the driver to all method e.g

Do this 
LanguageAndSkillsPage : CommonDriver

Or 

private IWebDriver driver;

Public LanguageAndSkillsPage(IWebDriver driver){
   this. driver = driver;
}


Instead of this? (don’t pass the driver this way )

 public void CreateLanguage(IWebDriver driver) {


}